AutomatedTesting/ShaderLib/scenesrg.srgi

```diff
@@ -14,15 +14,7 @@
 
 #include <Atom/Features/SrgSemantics.azsli>
 
-#define AZ_COLLECTING_PARTIAL_SRG_INCLUDES
```

Comment:
Is it ok if the deleted sections exist in older projects?

Reply:
If people have already updated their project to have these include sections will this solution work for them?

Reply:
yes, this will work in both cases, since the files included there have a `#pragma once`.